### DISCUSSION TITLE
[@types/three] ViewHelper Usage

### DISCUSSION THREAD

**Discussion by junjun1023** [DefinitelyTyped/DefinitelyTyped#66574] [Issues with a @types package]
Hi, I'm confused the correct way to use `ViewHelper`.

Here is my snippet
```typescript
        const scene = new THREE.Scene();
        const camera = new THREE.PerspectiveCamera(75, 1, 0.1, 1000);
        const renderer = new THREE.WebGLRenderer({ antialias: true });
        // ... scene setting up
       
        const viewHelper = new ViewHelper( camera, renderer.domElement ); 
        scene.add(viewHelper);
        
	const animate = () => {
		requestAnimationFrame(animate);

		if (viewHelper.animating) {
			const delta = clock.getDelta();
			viewHelper.update(delta);
		}

                trackballControl.update();
		renderer.render(scene, camera);
		viewHelper.render(renderer);
	};
```

My scene works like image below without `ViewHelper`
![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/44397290/92a2c0ee-66a2-451e-a141-c0a690fd6131)

As long as I create `ViewHelper` and called `viewHelper.render(renderer)`, objects in scene will disappear like image below.
![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/44397290/8a9c87dc-1757-414d-8c07-df94cc760d11)

Here is the result that I create `ViewHelper` without call the function `viewHelper.render(renderer)`
![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/44397290/48fe96e2-07f5-4d19-9fcc-1543f0713372)
I can't interact with `viewHelper`, and it is not located at bottom right .

Can anyone help me for the correct way to use `ViewHelper`?

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment]
Hi @junjun1023! This seems like more of a question about how to use three.js than about @types/three. I would suggest asking for help through one of the community links on [the main three.js site](https://threejs.org/).
